spark/arctern_spark/geodataframe.py

```diff
@@ -388,7 +479,7 @@ def reset_index(self, level=None, drop=False, inplace=False, col_level=0, col_fi
     @classmethod
     def from_file(cls, filename, **kwargs):
         """
-        Constructs a GeoDataFrame from a file or url.
+        Construct a GeoDataFrame from a file or url.
```

Comment:
The subject is "the function", so should add a 's' to all verbs. Please change the words below.